module/zfs/dbuf.c

```diff
@@ -687,6 +785,33 @@ dbuf_init(void)
 	cv_init(&dbuf_evict_cv, NULL, CV_DEFAULT, NULL);
 	dbuf_cache_evict_thread = thread_create(NULL, 0, dbuf_evict_thread,
 	    NULL, 0, &p0, TS_RUN, minclsyspri);
+
+	dbuf_ksp = kstat_create("zfs", 0, "dbuf_stats", "misc",
+	    KSTAT_TYPE_NAMED, sizeof (dbuf_stats) / sizeof (kstat_named_t),
+	    KSTAT_FLAG_VIRTUAL);
+	if (dbuf_ksp != NULL) {
```

Comment:
You could do:
```
if (dbuf_ksp == NULL)
    return;
```
...and then you wouldn't need to indent the rest of the code below.

Reply:
I thought about something like that, but I felt like having a block like that should be reserved for a condition that should be considered an "early exit" to a function (i.e. exits on errors). `dbuf_ksp` coming back as `NULL`, I don't believe, should be considered a terminal type condition.